src/openloco/GameCommands.cpp

```diff
@@ -242,10 +240,8 @@ namespace openloco::game_commands
 
                 case element_type::road: //0x1C
                 {
-                    road_element* roadElement = tile->as_road();
-                    assert(roadElement != nullptr);
-
-                    road_object* pObject = objectmgr::get<road_object>(roadElement->road_object_id());
+                    road_element& roadElement = *tile->as_road();
+                    road_object* pObject = objectmgr::get<road_object>(roadElement.road_object_id());
```

Comment:
NB: if missing, this will return a `-1` pointer rather than a `nullptr`. You probably want to change the relevant function in the `objectmgr`, like I did for land_object and scenario_text_object.